README.md

```diff
@@ -44,6 +44,26 @@ https://cdnjs.com/libraries/jquery.mask
 ### JSDelivr
 http://www.jsdelivr.com/projects/jquery.mask
 
+## RubyGems
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'jquery_mask_rails', '~> 0.1.0'
+```
+
+And then execute:
```

Comment:
I don't think that all those instructions are necessary. If they want to use they can locate the gem's owner and then find further instructions.